@here/harp-map-controls/lib/MapControls.ts

```diff
@@ -363,6 +372,9 @@ export class MapControls extends THREE.EventDispatcher {
      * @param deltaAltitude Delta altitude in degrees.
      */
     orbitFocusPoint(deltaAzimuth: number, deltaAltitude: number) {
+        if (this.mapView.projection.type === geoUtils.ProjectionType.Spherical) {
+            return;
```

Comment:
Why early return here? Please add a comment to the typedoc that this method will do nothing for globe.

Reply:
It should do something, it's only that it is not supported yet. It should be implemented in further tickets.